### ISSUE TITLE
Sync Light and Dark theme with macOS Dark Mode

### ISSUE THREAD

**Issue by diegobit** [sfsam/Itsycal#75]
It would be nice to be able to toggle macOS dark mode and to have Itsycal change it by itself.

**Comment by buffaloseven** [comment]
I agree it would be nice; in the meantime, I've hacked a solution together.

I use a workflow in Alfred app that runs an AppleScript when triggered by a hotkey to swap between macOS light/dark themes.  I've simply added a couple lines to change the Itsycal theme at the same time:

```as
	do shell script "defaults write com.mowglii.ItsycalApp ThemeIndex -int 0"
```

...sets Itsycal to the light theme and...

```as
	do shell script "defaults write com.mowglii.ItsycalApp ThemeIndex -int 1"
```

...sets it to the dark theme.  I've coordinated this with the OS theme swapping and it all matches nicely.

Obviously, this only works if triggered by the hotkey, but that's so convenient I never go into the system settings any more to swap between them.
